### DISCUSSION TITLE
`The text is currently under development`

### DISCUSSION THREAD

**Discussion by ghost** [pascalabcnet/pascalabcnet#3018] [General]
`Атрибуты` of `Классы` is not yet written.

**Comment by SunSerega** [comment]
Are they? Maybe you have an older version of help?

![image](https://github.com/pascalabcnet/pascalabcnet/assets/27270190/70f5626b-5c5a-4789-9221-20fc1382c7a2)

**Reply by SunSerega** [reply]
Tho I also don't understand what is that `След` at the end. Probably `Следующее` (next).

**Reply by ghost** [reply]
I'm using the online version: https://pascalabc.net/downloads/pabcnethelp/index.htm

It have to use it to use the machine translator. It's a Chrome extension.

**Reply by SunSerega** [reply]
Well, I asked devs to update it (a few mins ago), but I recommend you instead install DeepL - it's better at translating and has features to make the process easier (like translating selected text in any app with key-combo).
